recipe/meta.yaml

```diff
@@ -1,12 +1,14 @@
-{% set version = "21.2" %}
+{% set version = "4.21.1" %}
+{% set minor_micro = ".".join(version.split(".")[1:]) %}
+{% set libprotobuf_version = "3." + minor_micro %}
 
 package:
   name: protobuf
   version: {{ version }}
 
 source:
-  url: https://github.com/protocolbuffers/protobuf/archive/v{{ version }}/protobuf-v{{ version }}.tar.gz
-  sha256: ca983c9d2c8f8c935513642bcc4b2cbc64e4046e0bb16bf2ff893128577ece8c
+  url: https://github.com/protocolbuffers/protobuf/archive/v{{ minor_micro }}/protobuf-v{{ minor_micro }}.tar.gz
```

Comment:
As I mentioned in https://github.com/conda-forge/libprotobuf-feedstock/pull/123, I'm afraid that not having version in the URL might mean the bot doesn't detect future releases automatically. I have made myself a watcher of protobuf releases so I should be able to take care of them manually if that's the case.